coins.json

```diff
@@ -660,5 +660,16 @@
         "curve": "ed25519",
         "publicKeyType": "curve25519",
         "explorer": "https://wavesexplorer.com/tx/"
+    },
+    {
+        "id": "aeternity",
+        "name": "Aeternity",
+        "symbol": "AE",
+        "decimals": 18,
+        "blockchain": "Aeternity",
+        "derivationPath": "m/44'/457'/0'/0",
```

Comment:
Did you cross verify with ledger or trezor? same phrases -> same address

Reply:
Should I implement a special test case for it, or just check manually?

Reply:
both will work

Reply:
Good catch, indeed it was wrong. Correct is ` m/44'/457'/0'/0'/0'`. Cross checked with Aeternity base and Airgap vault apps. Added c++ test
Thx!